packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -50,7 +50,8 @@ let initialDocumentOverflowStyle: string = "";
 
 /**
  * @slot - A slot for adding content.
- * @slot content - A slot for adding custom content.
+ * @slot content - [Deprecated] Use `custom-content` slot instead.
```

Comment:
@DitwanP we would actually need to add this slot in the code. It not just a doc change. 

I'm kinda unsure how this change would work without being breaking since we put the panel inside the `content` slot by default.

Reply:
@driskull Yeah good point. That being said, what are your thoughts on changing the slot to the new one overall? Are you in favor or not in favor of such a change?

Reply:
I'm in favor of it, i'm just not sure how it will work exactly.

Reply:
Can this coincide with the rest of 3.0 breaking changes?

Reply:
Thats a question for @jcfranco

Reply:
Good catch. Unfortunately, it's too late to include in 3.x. Could we update the description for this release and schedule the slot rename for 4.0? cc @geospatialem

Reply:
Could we deprecate (w/o removing) "content" slot and add "custom-content"? That wouldn't be breaking as the recommendation is to use the "unnamed" default slot, right?

Reply:
The problem is we put default content in the `content` slot now. So we don't have a good way to counter that.

Reply:
Basically, we can't have this in both the `content` and `custom-content` slots or it would show up twice. 

```tsx
   <slot name={SLOTS.content}>
            <calcite-panel
              beforeClose={this.beforeClose}
              class={CSS.panel}
              closable={!this.closeDisabled}
              closed={!opened}
```

I don't think we can do this right?

```tsx
<slot name="custom-content">
            <slot name={SLOTS.content}>
```

Reply:
That... does seem to work pretty well, in local testing...

Reply:
Well i'll be... 🤯